images/Dockerfile.ubuntu

```diff
@@ -80,6 +75,61 @@ RUN apt-get update \
     zstd \
     && apt-get remove -y unattended-upgrades && apt-get clean && rm -rf /var/lib/apt/lists/*
 
+# Packages and scripts only for specific images
```

Comment:
my guess is that most of the files that are not consistent accross the Dockerfiles on master, are not because they are specifically required but because we failed to keep them in sync. But even with those differences, you can see how this section is actually pretty small and makes it easy to eval the differences between distro versions.